src/components/ChartContainer/D3Chart.js

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable no-underscore-dangle */
+import { TickChange } from 'components/Utilites/TickChange';
```

Comment:
The linter is complaining about this line.  The imports should look more like this

`import * as d3 from 'd3';`
`import PropTypes from 'prop-types';`
`import React, { useEffect, useRef, useState } from 'react';`
`import { TickChange } from '../Utilites/TickChange';`
`import { colorMappingProps } from './D3Props';`